src/themes/teams/components/Image/imageStyles.ts

```diff
@@ -4,11 +4,11 @@ export default {
   root: ({ props, variables }) => ({
     display: 'inline-block',
     verticalAlign: 'middle',
-    width: variables.width || (props.fluid && '100%'),
```

Comment:
this, actually, imposes other problem. Variables are more specific than props (in terms of the styling aspects targeting), so for the following case where conflicting things are requested

```jsx
<Image fluid variables={{ width: '100px' }} />
```

the result should be that Image will have `100px` width. Essentially, the following principle should be preserved: more specific customization option should always win. With the proposed change it won't be the case anymore

Reply:
I agree for the width, will change it, but for the avatar then we must provide something like `variables.avatarSize || (props.fluid && '100%') || '32px'` and we can have that variable to be `undefined` by default.

Reply:
After discussing, the decision was to have the prop be the first statement in the condition, because otherwise if the theme is specifying the default width, the fluid prop will never be correctly applied.